src/app/home/home.component.ts

```diff
@@ -19,13 +19,23 @@ export class HomeComponent implements OnInit {
   
   highlightCards: readonly HighlightCard[] = [
     {
-      headerText: 'Upcoming Match Funding Campaigns',
-      bodyText: "Discover our exciting range of match funding opportunities for 2023/2024",
-      iconColor: "tertiary",
-      backgroundImageUrl: new URL('/assets/images/peach-texture.jpg', environment.donateGlobalUriPrefix),
+      headerText: 'Save the date for Women and Girls Match Fund',
+      bodyText: "11th - 18th October 2023",
+      iconColor: "brand-2",
```

Comment:
brand-2 is deprecated

Reply:
We can update the highlight card type to use the brandColour type from the components library like so:

```diff
diff --git a/src/app/home/HighlightCard.ts b/src/app/home/HighlightCard.ts
index 97817063..c31da377 100644
--- a/src/app/home/HighlightCard.ts
+++ b/src/app/home/HighlightCard.ts
@@ -1,10 +1,10 @@
+import {brandColour} from "@biggive/components/dist/types/globals/brand-colour";
+
 export type HighlightCard = {
   backgroundImageUrl: URL,
-  // There is ambiguity in the components library about whether brand-6 is grey or turquoise. Best to avoid using brand-6 and
-  // use brand-mhf-turquoise instead.
   // See https://github.com/thebiggive/donate-frontend/pull/1076#issuecomment-1523472452
   // For other colour details see https://github.com/thebiggive/components/blob/develop/src/globals/brand-colour.ts
-  iconColor: 'primary' | 'tertiary' | 'brand-1' | 'brand-2' | 'brand-3' | 'brand-4' | 'brand-5' | 'brand-6' | 'brand-mhf-turquoise',
+  iconColor: brandColour
```

Reply:
For some reason I'm getting a compile error unless I bring the entire type to home component like in my last commit

Reply:
What's the error? I can take a look on a call maybe?